CONTRIBUTING.md

```diff
@@ -20,8 +20,8 @@ You can open a new issue with this [issue form](https://github.com/novuhq/novu/i
 
 ### Requirements
 
-- Node.js version v16.15.1
-- MongoDB
+- [Node.js](https://nodejs.org/en) 
```

Comment:
This should actually reference nvm-windows or nvm for linux or macos.

Can you add where we recommend the latest Node.js version of v16 install through nvm in some way and create documentation for that?

Reply:
let me fix that

Reply:
> This should actually reference nvm-windows or nvm for linux or macos.
> 
> Can you add where we recommend the latest Node.js version of v16 install through nvm in some way and create documentation for that?

Can you please check now ?